conda/cli/main_create.py

```diff
@@ -129,14 +129,17 @@ def execute(args: Namespace, parser: ArgumentParser) -> int:
                 "Cannot `create --dry-run` with an existing conda environment"
             )
         confirm_yn(
-            f"WARNING: A conda environment already exists at '{context.target_prefix}'\n"
-            "Remove existing environment",
+            f"WARNING: A conda environment already exists at '{context.target_prefix}'\n\n"
+            "Remove existing environment?\nThis will remove ALL directories contained within "
+            "this specified prefix directory, including any other conda environments.\n\n",
             default="no",
             dry_run=False,
         )
-        log.info("Removing existing environment %s", context.target_prefix)
+        log.info(f"Removing existing environment {context.target_prefix}")
```

Comment:
We prefer to leave logging as `%` style formatting to avoid unnecessary templating when the log level results in the message not being printed:

```suggestion
        log.info("Removing existing environment %s", context.target_prefix)
```